content/pt-br/_collections/navigations/footer.md

```diff
@@ -0,0 +1,55 @@
+---
+links:
+  - title: 'Sobre'
+    items:
+      - title: 'Fale conosco'
+        href: 'mailto:hello@nuxtjs.com'
+      - title: 'Enterprise support'
```

Comment:
`Supporte para empresas`?

Reply:
Yes, that is correct, I've made the change.